src/formio/index.ts

```diff
@@ -71,11 +73,14 @@ export type AnyComponentSchema =
   | BsnComponentSchema
   | AddressNLComponentSchema
   | NpFamilyMembersComponentSchema
+  | CosignV2ComponentSchema
   | MapComponentSchema
   // layout
   | ContentComponentSchema
   | ColumnsComponentSchema
-  | FieldsetComponentSchema;
+  | FieldsetComponentSchema
+  // deprecated
+  | CosignV1ComponentSchema;
```

Comment:
Should the other deprecated components move here as well? (eg postcode)

Reply:
yeah, I'll make the change after everything is merged, got enough merge conflicts already as it is